workflows/metatdenovo.nf

```diff
@@ -100,19 +97,22 @@ include { FASTQC_TRIMGALORE } from '../subworkflows/local/fastqc_trimgalore'
 include { DIGINORM } from '../subworkflows/local/diginorm'
 
 //
+// Is this really a good heading? I assume it's taken from the example above (INPUT_CHECK), but does one really need to know what's in the subworkflow? Or should we collect all subworkflows under the possible three headings: only nf-core, only local, mix?
```

Comment:
Just saw this comment. we could probably just collect the subworkflows under those three headings. I'm not sure about the guidelines but it should be fine